frontend/device/kobo/nickel_conf.lua

```diff
@@ -150,46 +181,76 @@ function NickelConf._write_kobo_conf(re_Match, key, value, dont_create)
     return true
 end
 
+--[[--
+Set frontlight level.
+
+@int new_intensity
+--]]
 function NickelConf.frontLightLevel.set(new_intensity)
-    assert(new_intensity >= 0 and new_intensity <= 100,
-           "Wrong brightness value given!")
+    if type(new_intensity) ~= "number" or (not (new_intensity >= 0) and (not new_intensity <= 100)) then return end
```

Comment:
I should've probably split this up into two or three commits. :man_facepalming: Anyway, this line here and the matching line below + moving the assert to dbg:guard are the meat of this thing, plus that one line in powerd. The docs make the diff look a lot more imposing than it is.